ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -466,7 +468,7 @@ public void notifyTerminated() {
   public void shutdown() {
     log.info("ksqlDB shutdown called");
     try {
-      streamedQueryResource.closeMetrics();
+      pullQueryExecutor.closeMetrics();
```

Comment:
What about the existing `streamedQueryResource.closeMetrics()` call?

Reply:
I moved the method to `pullQueryExecutor` and then just call that.  Same logic being called, just in a different place.